components/eam/cime_config/config_compsets.xml

```diff
@@ -130,54 +130,10 @@
 
   <!-- MMF aquaplanet compsets -->
   <compset>
-    <alias>F-MMF1-AQP1</alias>
+    <alias>F-AQUA-MMF1</alias>
```

Comment:
I don't like this change. The convention with the rest of the EAM compsets is `F-EAM-<modifier>`, so the aquaplanet configuration would be `F-EAM-AQP`. Here, you are changing the MMF one to be `F-<modifier>-MMF`, which is also contrary to the full compset spec below (`2000_EAM%MMF1-AQUA`).